tutorial/README.md

```diff
@@ -0,0 +1,391 @@
+
+if __name__ == "__main__":
+    val = my_b_task.submit("Agrajag")
+    print(val)
+```
+
+Step 4: Open another terminal and run the script.
+
+```bash
+python task_submitter.py
+```
+
+**TIP:** For Celery users, `Task.submit()` is similar to `Task.delay()` or
+`Task.apply_async()`. For arq users, it's similar to `Task.enqueue()`.
+
+Note that we return the task run object from the `submit` method.
+This way we can see the task run UUID and other information about the task run.
+
+Step 5: See the task run in the UI.
+
+Use the task run UUID to see the task run in the UI.
+The URL will look like this:
+
+<http://127.0.0.1:4200/task-runs/task-run/my_task_run_uuid_goes_here>
+
+Substitute your UUID at the end of the URL.
+Note that the UI navigation experience for task runs will be improved soon.
+
+Step 6: You can use multiple task servers to run tasks in parallel.
+
+Start another instance of the task server.
+
+In another terminal run:
+
+```bash
+python task_server.py
+```
+
+Step 7: Submit multiple tasks to the task server with `map`.
+
+Modify the `task_submitter.py` file to submit multiple tasks to the task server with different inputs by using the `map` method.
+
+```python
+from task_server import my_b_task
+
+if __name__ == "__main__":
+    my_b_task.map(["Ford", "Prefect", "Slartibartfast"])
+```
+
+Run the file and watch the work get distributed across both task servers!
+
+Step 8: Shut down the task servers with *control* + *c*
+
+Alright, you're able to submit tasks to multiple Prefect task servers running in the background!
+This is cool because we can observe these tasks executing in parallel and very quickly with web sockets - no polling required.
+
+Next, let's wire up our task server to a FastAPI task server.
+
+## Example 3: Create a basic FastAPI server that submits tasks to a Prefect task server
+
+Step 1: Define API routes for the FastAPI server in a Python file.
+
+Let's define two routes for our FastAPI server.
+The first is a basic hello world route at the root URL to confirm that the FastAPI server is working.
+The second route, `/task`, will submit a task to the Prefect task server when the URL is hit and return information about the submitted task.
+You could name this route whatever you like.
+
+Here are the contents of [first_fastapi.py](./tutorial/first_fastapi.py)
+
+```python
+from fastapi import FastAPI
+from prefect import task
+from first_fastapi_task_server import my_fastapi_task
+
+app = FastAPI()
+
+
+@task
+def my_b_task(name: str):
+    print(f"Hello, {name}!")
+    return f"Hello, {name}!"
+
+
+@app.get("/")
+async def root():
+    return {"message": "Hello World"}
+
+
+@app.get("/task")
+async def prefect_task():
+    val = my_fastapi_task.submit(name="Marvin")
+    return {"message": f"Prefect Task submitted: {val}"}
+```
+
+Step 2: Define a Prefect task server in a Python file.
+
+Here are the contents of [ff_prefect_task_server.py](./tutorial/ff_prefect_task_server.py)
```

Comment:
can we use a better name than `ff_prefect_task_server`? i dont know what `ff` refers to

Reply:
yeah. lol. That was bugging me.